manim/mobject/graphing/coordinate_systems.py

```diff
@@ -2412,6 +2466,59 @@ def construct(self):
         positioned_label.rotate(rotation, axis=rotation_axis)
         return positioned_label
 
+    def get_axis_labels(
+        self,
+        x_label: float | str | Mobject = "x",
+        y_label: float | str | Mobject = "y",
+        z_label: float | str | Mobject = "z",
+    ) -> VGroup:
+        """Defines labels for the x_axis and y_axis of the graph.
+
+        For increased control over the position of the labels,
+        use :meth:`get_x_axis_label` and :meth:`get_y_axis_label`.
+
+        Parameters
+        ----------
+        x_label
+            The label for the x_axis. Defaults to :class:`~.MathTex` for ``str`` and ``float`` inputs.
+        y_label
+            The label for the y_axis. Defaults to :class:`~.MathTex` for ``str`` and ``float`` inputs.
+        z_label
+            The label for the z_axis. Defaults to :class:`~.MathTex` for ``str`` and ``float`` inputs.
+
+        Returns
+        -------
+        :class:`~.VGroup`
+            A :class:`~.Vgroup` of the labels for the x_axis, y_axis and z_axis.
```

Comment:
The Oxford comma is debatable, but the spelling of VGroup should definitely be corrected :-)

```suggestion
            A :class:`~.VGroup` of the labels for the x_axis, y_axis, and z_axis.
```